docs/eventing/samples/writing-ra-source/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Writing an event source from scratch"
+linkTitle: "Writing an event source from scratch"
```

Comment:
```suggestion
```

Reply:
No need to use a linkTitle if it's identical to the full title. This is useful if the title is very long and would be hard to read in the sidebar; linkTitle controls the name that shows up in the sidebar.

Reply:
Done